### DISCUSSION TITLE
Is this "AI" system ok?

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2526] [Q&A]
![image](https://user-images.githubusercontent.com/60806448/161080124-f1297b7f-1d09-47c8-ba24-baa3f44183e3.png)
I call the move function one time in "create" function of a state this Enemy script is present in, and then each time that should be called automatically.. on paper
(after compiling, the enemy just isn't seeable for some reason)

**Comment by RichardBray** [comment]
To be honest, if it works for you it works, don't stress about it too much.

If you're looking to make it better I would look into [finite state machines](https://www.youtube.com/watch?v=z3qQpGHozso) and behaviour trees.
